rstudio-tidyverse/rockcraft.yaml

```
@@ -0,0 +1,259 @@
+# Based on:
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/rstudio-tidyverse/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/rstudio/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/base/Dockerfile
+
+name: rstudio-tidyverse
+summary: An image for using tidyverse
+description: |
+  The tidyverse is an opinionated collection of R packages designed for data science. 
+  Tidyverse are installed in Conda environment, which is automatically activated.
+version: "v1.10.0-rc.1"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+services:
+  rstudio-tidyverse:
+    override: replace
+    summary: "Rstudio Tidyverse service"
+    startup: enabled
+    command: /usr/lib/rstudio-server/bin/rserver --server-daemonize=0 --server-data-dir=/tmp/rstudio_runtime --server-working-dir=/home/_daemon_ --server-user=_daemon_ --rsession-which-r=/opt/conda/lib/R --www-address=0.0.0.0 --www-port=8888 --auth-none=1 --rsession-path=/opt/rsession.sh --rsession-ld-library-path=/opt/conda/lib
```

suggestion(blocking): Let's use the `jovyan` user
```suggestion
    command: /usr/lib/rstudio-server/bin/rserver --server-daemonize=0 --server-data-dir=/tmp/rstudio_runtime --server-working-dir=$HOME --server-user=$NB_USER--rsession-which-r=/opt/conda/lib/R --www-address=0.0.0.0 --www-port=8888 --auth-none=1 --rsession-path=/opt/rsession.sh --rsession-ld-library-path=/opt/conda/lib
```